src/store/todo/todo.selectors.ts

```diff
@@ -0,0 +1,68 @@
+import { SortTypes, TodoDTO } from '~types/todo.types';
+import { stringSorting } from '~utils/stringSorting';
+import { booleanSorting } from '~utils/booleanSorting';
+import { AppState } from '~store';
+
+const loading = (state: AppState): boolean => state.todo.loading;
+const error = (state: AppState): string => state.todo.error;
+const cursor = (state: AppState): number => state.todo.cursor;
+const editingMode = (state: AppState): boolean => state.todo.editingMode;
+const editingTodos = (state: AppState): Array<TodoDTO> => state.todo.editingTodos;
+const editingInput = (state: AppState): string => state.todo.editingInput;
+const filterMode = (state: AppState): SortTypes => state.todo.filterMode;
+const allTodosCount = (state: AppState): number => state.todo.allTodosCount;
+const isListInitializing = (state: AppState): boolean =>
+  state.todo.todos.length === 0 && state.todo.loading;
+const page = (state: AppState): number =>
+  state.todo.skip ? state.todo.skip / state.todo.limit + 1 : 1;
+const todos = (state: AppState): Array<TodoDTO> => {
+  const todosList = [...state.todo.todos];
+  const edidtinTodos = state.todo.editingTodos;
+  const notEditingTodos = todosList.filter(
+    (todo) => !edidtinTodos.some((eTodo) => eTodo.id === todo.id),
+  );
+
+  switch (state.todo.filterMode) {
+    case SortTypes.ID_ASC:
```

Comment:
I would also move cases like ID_... to the separate files, let's investigate if its possible.

Reply:
Moved sorting logic for each sortType group (like ID...) to separate files